sky/backends/backend_utils.py

```diff
@@ -958,7 +957,6 @@ def write_cluster_config(
 
                 # Kubernetes only:
                 'skypilot_k8s_image': k8s_image,
-                'ssh_key_secret_name': ssh_key_secret_name,
```

Comment:
`ssh_key_secret_name` doesn't seem to be used.

Reply:
Ah thanks for catching - this is removed in the current latest k8s_cloud branch.